### PR TITLE
Make a bower package, and uglify with gulp.

### PR DESCRIPTION
**Pull request by MichaelJCole**
Hey, I wanted this as a bower.io package.  Since I don't grok the code, it would be better as a PR than a fork.

Thanks BTW!  Mike

Changelog:
Add bower.json to make a bower.io package. 
Add gulp build tool to uglify into dist/base32.min.js. 
Add tests for minified code.

### CODE REVIEW

**Comment by MichaelJCole** [comment]
Here's how to push the package to bower.io
http://bower.io/docs/creating-packages/#register

**Comment by MichaelJCole** [comment]
Closing to retry Travis.

**Comment by MichaelJCole** [comment]
The Travis error is re: gulp-load-plugins not working in Nodejs 0.8.  It's an issue with the minification, so we could:

  - ignore it.  Don't build on 0.8
  - fix it.  Declare the plugins directly.

I think it depends on if you want this PR or not.  Lemme know what is best and I'll fix it up!

Note: If you like, you can re-run the minification yourself by running 'npm install && gulp'.

Mike

**Comment by agnoster** [comment]
I say we can ignore 0.8 at this point, so I'd be up for removing that as a supported target

**Comment by MichaelJCole** [comment]
The light is green, the trap is clean :-)

File: test/base32min-test.coffee [hunk 1]
```diff
@@ -0,0 +1,76 @@
+vows = require 'vows'
+assert = require 'assert'
+base32 = require '../dist/base32.min.js'
+crypto = require 'crypto'
+
+suite = vows.describe 'Base32 Encoding'
```
Comment (agnoster) [inline test/base32min-test.coffee]: We should really not duplicate the actual code for the tests, and instead have one include for the shared suites, and then run separate tests over the unminified and minified version.

But I guess that can happen in a later PR :-)

**Pull request merged**